mriqc/workflows/utils.py

```diff
@@ -35,72 +35,3 @@ def fwhm_dict(fwhm):
     fwhm = [float(f) for f in fwhm]
     return {'x': fwhm[0], 'y': fwhm[1],
             'z': fwhm[2], 'avg': fwhm[3]}
-
-def fd_jenkinson(in_file, rmax=80., out_file=None):
```

Comment:
why removing this? I don't think ```3dvolreg``` outputs parameters according to the FD definition. This conversion might be broken, but I don't think removing this is a good idea.